Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:16-alpine AS development
```

Comment:
Again, Node v20 would be preferred.

Additionally, is there a reason why you're using an Alpine image? Alpine can be the cause of [many issues](https://martinheinz.dev/blog/92).